modules/snooker/src/main/ts/ephox/snooker/util/CellUtils.ts

```diff
@@ -1,14 +1,19 @@
 import { Fun } from '@ephox/katamari';
-import { Attribute, Css, SugarElement } from '@ephox/sugar';
+import { Attribute, Css, SugarElement, SugarNode } from '@ephox/sugar';
 
 export const getAttrValue = (cell: SugarElement<Element>, name: string, fallback: number = 0): number =>
   Attribute.getOpt(cell, name).map((value) => parseInt(value, 10)).getOr(fallback);
 
 export const getSpan = (cell: SugarElement<HTMLTableCellElement>, type: 'colspan' | 'rowspan'): number =>
   getAttrValue(cell, type, 1);
 
-export const hasColspan = (cell: SugarElement<HTMLTableCellElement>): boolean =>
-  getSpan(cell, 'colspan') > 1;
+export const hasColspan = (cell: SugarElement<HTMLTableCellElement | HTMLTableColElement>): boolean => {
```

Comment:
Can we rename this to `elem` or similar now that it could also be a `col`?

Reply:
Renamed it to `cellOrCol`.